specification/personal-demographics.yaml

```diff
@@ -489,19 +497,21 @@ paths:
-        | Unsuccessful search         | Any other valid search query parameters                      | HTTP Status 200 with no search results     |
-        | Invalid date format         | `birthdate` or `death-date` query parameters with values not in format `YYYY-MM-DD` | HTTP Status 400 with problem description   |
-        | Too few search parameters   | Any invalid combination of query parameters                  | HTTP Status 400 with problem description   |
-        | Missing X-Request-ID        | `id`=`9000000009` (or any other valid NHS number)            | HTTP Status 412 with problem description   |
+        | Scenario                                | Request                                                                                                                                                         | Response                                                                |
+        | ----------------------------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------------------| ------------------------------------------------------------------------|
+        | Basic search                            | `family`=`Smith`, `gender`=`female`, `birthdate`=`eq2010-10-22`, `email`=`jane.smith@example.com`, `phone`=`01632960587`                                        | HTTP Status 200 with single search result.                              |
+        | Wildcard search                         | `family`=`Sm*`, `gender`=`female`, `birthdate`=`eq2010-10-22`                                                                                                   | HTTP Status 200 with search result of two.                              |
+        | Wildcard search with email and phone    | `family`=`Sm*`, `gender`=`female`, `birthdate`=`eq2010-10-22`, `email`=`jane.smith@example.com`, `phone`=`01632960587`                                          | HTTP Status 200 with single search result.                              |
+        | Search with limited results             | `family`=`Sm*`, `gender`=`female`, `birthdate`=`eq2010-10-22`, `email`=`jane.smith@example.com`, `phone`=`01632960587`, `_max-results`=`1`                      | HTTP Status 200 with single search result.                              |
+        | Search with date range                  | `family`=`Smith`, `gender`=`female`, `birthdate`=`ge2010-10-21`, `birthdate`=`le2010-10-23`, `email`=`jane.smith@example.com`, `phone`=`01632960587`            | HTTP Status 200 with single search result.                              |
+        | Fuzzy search                            | `family`=`Smith`, `given`=`Jane`, `gender`=`female`, `birthdate`=`eq2010-10-22`, `email`=`jane.smith@example.com`, `phone`=`01632960587`, `_fuzzy-match`=`true` | HTTP Status 200 with single search result.                              |
```

Comment:
was just looking. I think there are two potential cases for fuzzy search. Without email/phone, it will return Smythe, with email/phone filtering, I'd think it would return Smith.

Reply:
added additional fuzzy search case without email/phone